### ISSUE TITLE
Compile data - Foreign Catches

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [pfmc-assessments/canary_2023#48]
The Rogers_foreign_catch (2003) pdf in the google drive contains the estimates of canary for foreign fleets. Need to add within our landings script.

**Comment by brianlangseth-NOAA** [comment]
The 2015 base model file actually has UVAN **and COL** as Washington, **EUR** as Oregon, and MON as California which differs from their catch master file, which uses the designation as I describe in the comment above. Previous assessments (really the 2002 assessment, because the 2005 and 2007 versions copied the 2002 choices) used the same as Im using here. Plan to keep with what Ive already done, which will differ slightly with the 2015 assessment.

**Comment by okenk** [comment]
Thanks Whitney, comment success!

Unfortunately, this is a big issue the historical catches between Oregon and Washington for both domestic and foreign fleets, and part of the reason we prefer a single area model for this cycle, since neither agency has the resources to split them up, and it is not clear if it can even be done in a reasonable way. The most important thing is these catches get assigned to *a* state, and it sounds like you think the majority of the landings would be off the coast of Oregon, not Washington. Although we may estimate different size selectivity curves for Oregon and Washington, they are not expected to be wildly different, so the state where these catches get assigned is unlikely to matter too much for the coastwide status. 

However, if you prefer, we could assign 20% of the COL catches to Washington. Pinging @tsoutt if you have any thoughts?

**Comment by brianlangseth-NOAA** [comment]
Thanks for the comment @robwm477. You are right that catches in COL and EUR are not going to align perfectly with state boundaries so the assumptions we made are indeed simplifying. We could look at the earliest pacfin years to get a sense of proportions to allocate COL and EUR among the three states. Would assume that boats from each states only fish in their state's portion of the INPFC though, which we know is a tenuous assumption. Oregon data also only goes back to 1987 in pacfin. 

I happened to look for 1987-1996, and in COL, 94% of our TWL grouping is from Oregon boats. In EUR, the proportions are more variable and average 32% for Oregon boats in that same time span. Given all of the assumptions this entails, Im comfortable making the same generalized assumptions as previous assessments and assigning COL to Oregon, and EUR to California.
